packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -231,8 +237,18 @@ export class Accordion extends GOVUKFrontendComponent {
     const $heading = $header.querySelector(`.${this.sectionHeadingClass}`)
     const $summary = $header.querySelector(`.${this.sectionSummaryClass}`)
 
-    if (!$span || !$heading) {
-      return
+    if (!$heading) {
+      throw new ElementError({
+        componentName: 'Accordion',
+        identifier: `Section heading (\`.${this.sectionHeadingClass}\`)`
+      })
+    }
+
+    if (!$span) {
+      throw new ElementError({
+        componentName: 'Accordion',
+        identifier: `Section button (\`<span class="${this.sectionButtonClass}">\`)`
```

Comment:
**nitpick** Maybe a little pedantic, but it looked weird to have "button" followed by a `span` tag.

```suggestion
        identifier: `Section button placeholder (\`<span class="${this.sectionButtonClass}">\`)`
```